.github/workflows/edit_pr_description.yml

```diff
@@ -8,6 +8,7 @@ on:
       - synchronize
       - reopened
       - edited
+  workflow_call:
   # see https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
```

Comment:
This comment refers to the line below and it's just poorly indented, maybe de-indent for clarity (and if anything add a comment about this empty "workflow_call" field)

Reply:
done